### DISCUSSION TITLE
在想說，PCL如果加個光影包和材質包下載會不會更方便

### DISCUSSION THREAD

**Discussion by snowolf50161** [Meloong-Git/PCL#4489] [讨论]
這樣就能減少找不到光影和材質包資源的問題啦，不過很大的問題是大部分光影包和材質都散落在不同網站

**Comment by hejiehao** [comment]
光影：https://github.com/Hex-Dragon/PCL2/issues/396
材质（资源）包：https://github.com/Hex-Dragon/PCL2/issues/44
